### PR TITLE
bugs and seed data

### PR DESCRIPTION
**Pull request by kkoskelin**
added seed data document for 120-20 (will be used for sprint43 demo)
added more behavior plus tests for 'replaceBracketed' (witnessed console error, fixed on the spot)
missing test code that I meant to add to previous PR

### CODE REVIEW

File: web-client/src/presenter/actions/chooseWorkQueueAction.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    });
+
+    expect(result.state.workQueueToDisplay.queue).toEqual('something');
+    expect(providers.path.messagessomethinginbox).toHaveBeenCalled();
+  });
+
+  it('combines provided props.box with defaults and state to select correct work queue path', async () => {
+    const result = await runAction(chooseWorkQueueAction, {
+      props: {
+        box: 'cardboard',
```
Comment (matthopson) [inline web-client/src/presenter/actions/chooseWorkQueueAction.test.js]: This really cracked me up. Thank you.

File: web-client/src/presenter/actions/chooseWorkQueueAction.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      state: {
+        workQueueToDisplay: {
+          box: 'something',
+          workQueueIsInternal: false,
+        },
+      },
+    });
+
+    expect(result.state.workQueueToDisplay.box).toEqual('cardboard');
+    expect(providers.path.documentqcmycardboard).toHaveBeenCalled();
```
Comment (matthopson) [inline web-client/src/presenter/actions/chooseWorkQueueAction.test.js]: In these last two tests, should we actually be testing the combined path with the defaults?

Comment (kkoskelin) [reply web-client/src/presenter/actions/chooseWorkQueueAction.test.js]: I just merged develop here to fix these broken tests, where the action itself has its own non-exported defaults:
```
const DEFAULT_QUEUE_PREFS = {
  box: 'inbox',
  queue: 'my',
  workQueueIsInternal: true,
};
```
In this way, the default of `{queue: 'my'}` is being used in order to compose the workQueuePath.

**Review by matthopson — APPROVED** [review]

**Pull request merged**